docs/ReleaseNotes.md

```diff
@@ -1,289 +1,401 @@
+
+Starting with 2020.11, Enklu Cloud now supports direct imports for the following CAD formats:
+- .asm 
+- .sldasm, .sldasm.1 (in a .zip file with any dependencies)
+- .prt, .prt.1
+- .sldprt, .sldprt.1
+- .dwg
+- .dxf
+- .fbx
+- .obj
+- .pxz (in a .zip file with any dependencies)
+- .gltf
+- .3dm
+- .stl 
+
+And this is only the beginning! We'll be continuing to update Enklu Cloud to accommodate more and more CAD formats as time goes on. So if you use a format that isn't represented above, please [contact us](/contact)!
+
+![CAD Assets](/img/product/releases/2020.11/CAD_asset_support.gif)
+
+
+##### Better Video Support
+
+Aside from CAD support, we've also added better video support in 2020.11. You can now upload the following video format directly to Enklu Cloud:
+
+ - .asf
+ - .avi
+ - .dv
+ - .m4v
+ - .mov
+ - .mp4
+ - .mpg
+ - .mpeg
+ - .ogv
+ - .vp8
+ - .webm
+ - .wmv 
+
+You can also directly embed video URLs into your projects [using VineML](API/Vines.md#video).
+
+> **Note:** Videos currently will not play in the web editor, but they will play as expected on supported devices.
+
+
+![Better Video Support](/img/product/releases/2020.11/video_asset_support.gif)
+
+
+
+### Improvements
+
+#### My Experiences Screen
+
+In this update, we've revamped the entire My Experiences screen, making it faster than ever to find and create experiences! Not only can you create new projects and modify existing ones, but you can also search for them directly.
+
+We've also added user portraits on each experience card to tell you who has access to a file (you can check their names on mouseover). 
+
+Finally, we've also added the ability to sort your experiences by either Name or Recent Activity. No more are the days of spending minutes chasing down a file you just started working on!
+
+![My Experience Screen](/img/product/releases/2020.11/my_experiences_screen.gif)
+
+
+#### Drag & Drop Scripts
+
+We've added a bunch of new Public Scripts in 2020.11 that makes building interactions easier than ever! These Drag & Drop scripts allow even the least technical of folks to create complex simulations or immersive content. 
+
+To use one of these scripts, create a new asset and drag one of these  over to the Script section of the Inspector. You can then adjust different parameters, like the icon or color of a button, in the Variable window. Most importantly, you can also decide what message an interactable takes, and what message it sends on completion.  You could create a button that pulls apart a motor, or a proximity trigger that reveals a hidden message! 
+
+For more on Drag & Drop scripts, check out our official guide [here]().
+
+![Drag & Drop Scripts](/img/product/releases/2020.11/drag_and_drop_scripts.gif)
+
+
+#### Web Play Mode: Hand Interactions
+
+Building on its official debut in 2020.10, we've added a new feature to Web Play mode: hand interactions! You can now simulate grabbing particles (or fingerpainting) without needing to put on a HoloLens.

```

Comment:
I feel like this makes it seem like you can do more with the hand in web play mode than you actually can.  Because you cannot change the way the hand is posed, so you can't _actually_ grab anything.  Maybe it would be better to say something along the lines of "You now have access to simulate basic hand interactions directly in your browser without needing to put on a HoloLens."

Reply:
While Hands are the latest & greatest in limb based input, this feature also does make it so the Touch, and older Gestures API are usable in Web Play Mode too. So experiences like the circuit board exploder that only use Simple Touch no longer need separate test calls.